src/app/view/dialogs/trainrun-and-section-dialog/trainrunsection-tab/trainrun-section-tab.component.ts

```diff
@@ -138,7 +140,10 @@ export class TrainrunSectionTabComponent implements AfterViewInit, OnDestroy {
       .getTrainrun()
       .getTimeCategoryLinePatternRef();
     this.trainrunSectionTimesService.setHighlightTravelTimeElement(false);
-    this.numberOfStops = this.selectedTrainrunSection.getNumberOfStops();
+    this.numberOfStops = this.trainrunSectionService.getNumberOfCollapsedStops(
```

Comment:
Relying on `this.selectedTrainrunSection` isn't great here, but fixing this would be a much larger change (https://github.com/OpenRailAssociation/netzgrafik-editor-frontend/issues/679), so let's leave that for later.